### DISCUSSION TITLE
Github Pages in every PR

### DISCUSSION THREAD

**Discussion by fernandoreta** [community/community#58843] [Programming Help]
### Body

I think github Pages is a ver powerful tool, but can I use that in every PR that I have?
For example the github paage of my repo is
Www.github.io/demo
And im looking to create a separates ones with the number of my pr and obviusly the changes of the pr
Www.github.io/demo/pr1
Www.github.io/demo/pr2
…
Could someone help me here please :)

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by anhtran917** [comment]
Yes, you can use GitHub Pages to create separate websites for each pull request in your repository. 
To do this, create a new branch and folder for each pull request, copy your website files into each folder, and specify the branch and custom domain in the GitHub Pages settings. 
Repeat these steps for each pull request you want to create a separate website for.
